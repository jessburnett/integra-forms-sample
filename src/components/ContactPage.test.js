import React from 'react'
import { cleanup, render } from 'react-testing-library'

jest.mock('../containers/ContactForm', () => () => <h1>Home</h1>);

describe('Component => RenderField', () => {
  afterEach(() => {
    cleanup();
  });

  it('should match snapshot', () => {
    const ContactPage = require('./ContactPage').ContactPage;

    const { container } = render(<ContactPage/>);

    expect(container.firstChild).toMatchSnapshot();
  })
});