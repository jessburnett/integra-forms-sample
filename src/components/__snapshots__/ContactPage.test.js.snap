// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Component => RenderField should match snapshot 1`] = `
<h1>
  Home
</h1>
`;
